source/shared/cpp/AdaptiveCardsSharedModel/AdaptiveCardsSharedModelUnitTest/DateAndTimeUnitTest.cpp

```diff
@@ -1,6 +1,7 @@
 #include "stdafx.h"
 #include "CppUnitTest.h"
 #include "TextBlock.h"
+#include "MarkDownParser.h"
```

Comment:
This include doesn't seem necessary

Reply:
thanks!

---
In reply to: [154205464](https://github.com/Microsoft/AdaptiveCards/pull/1003#discussion_r154205464) [](ancestors = 154205464)